test/auro-card.test.js

```diff
@@ -25,4 +27,309 @@ describe('auro-card', () => {
 
     await expect(el).to.be.true;
   });
+
+  describe('Background color', async () => {
+    it('sets background color when given a background color', async () => {
```

Comment:
With a simpler API, we can really reduce the number of tests we need to support as well.